app/plans/get.controller.js

```diff
@@ -2,32 +2,46 @@ const { logger } = require('../../common/logging/logger')
 const { getSentencePlanSummary } = require('../../common/data/sentencePlanningApi')
 const { isEmptyObject } = require('../../common/utils/util')
 
-const getPlan = plans => {
+const getPlan = (plans = {}) => {
   if (isEmptyObject(plans)) return {}
   return (
     plans.find(({ completedDate }) => completedDate === '' || completedDate === undefined || completedDate === null) ||
     {}
   )
 }
 
+const getCompletedPlans = plans => {
+  if (isEmptyObject(plans)) return {}
+  return plans.filter(
+    ({ completedDate }) => completedDate !== '' && completedDate !== undefined && completedDate !== null
```

Comment:
This is kinda duplicated from the above.
```
const isValid =str => str !== '' && str !== undefined && str !== null
```
then call... from line 8
```
return plans.find(({ completedDate }) => !isValid(completedDate)) || {}
```
...and line 16
```
return plans.filter(({ completedDate }) => isValid(completedDate))
```